packages/ui/src/2_molecules/AmountInput/AmountInput.module.css

```diff
@@ -43,11 +43,19 @@
 }
 
 .invalid {
-  @apply outline outline-1 outline-error-light bg-gray-60;
+  @apply outline outline-1 outline-error-light bg-gray-70;
+
+  &:hover {
+    @apply outline-error-light;
```

Comment:
Invalid border color seems to take precedence here as it doesn't change when hovering in storybook, but I don't see this in Figma so I think this can just be removed.

Reply:
This was actually requested from Gilad because without this, you could hover over invalid/focused states and you'd see hover effect that shouldn't be there.